assignments/004-iteration-and-fastp/script.sh

```diff
@@ -15,17 +15,16 @@ mkdir -p results/trimmed results/qc/fastp
 # DPCh_plate1_B10_S22 DPCh_plate1_B11_S23 DPCh_plate1_B12_S24 DPCh_plate1_C10_S34 ...
 # WITH ALL THE SAMPLE NAME THERE
 
-SAMPLES=..your code goes here..
-
+SAMPLES=(DPCh_plate1_B10_S22 DPCh_plate1_B11_S23 DPCh_plate1_B12_S24 DPCh_plate1_C10_S34 DPCh_plate1_C11_S35 DPCh_plate1_C12_S36 DPCh_plate1_D09_S45 DPCh_plate1_D11_S47 DPCh_plate1_F10_S70 DPCh_plate1_F11_S71 DPCh_plate1_F12_S72 DPCh_plate1_G09_S81 DPCh_plate1_G10_S82 DPCh_plate1_G12_S84 DPCh_plate1_H09_S93 DPCh_plate1_H10_S94)
 
 # NOW, MODIFY THE FOLLOWING, USING VARIABLE SUBSTITUTION SO THAT S IS USED IN PLACE OF THE SAMPLE
 # NAME IN THE COMMAND (i.e. replace DPCh_plate1_B10_S22 with the variable S---not that you will
 # need to wrap S in curly braces!)
 
-for S in $SAMPLES; do
-    fastp -i data/fastqs/DPCh_plate1_B10_S22_R1.fq.gz -I data/fastqs/DPCh_plate1_B10_S22_R2.fq.gz  \
-          -o results/trimmed/DPCh_plate1_B10_S22_R1.fq.gz -O results/trimmed/DPCh_plate1_B10_S22_R2.fq.gz \
-          -h results/qc/fastp/DPCh_plate1_B10_S22.html  -j results/qc/fastp/DPCh_plate1_B10_S22.json \
+for S in ${SAMPLES[@]}; do
```

Comment:
Changed this from $SAMPLES because it was only acting on the first element in the array